openstack/neutron/templates/vct-nsxv3-nanny.yaml

```diff
@@ -18,6 +18,9 @@ template: |
       system: openstack
       type: backend
       component: neutron
+      vcenter: {= vcenter_name =}
```

Comment:
It's `{= vcenter_name =]` here, but `{= host =}` below. We have to make up our mind which one we take.

Reply:
I've changed it for now everywhere to `host`